examples/bri-3/src/bri/transactions/api/transactions.controller.spec.ts

```diff
@@ -106,41 +112,43 @@ describe('TransactionController', () => {
       const fromBpiSubjectAccount = await createBpiSubjectAccount('123');
       const toBpiSubjectAccount = await createBpiSubjectAccount('321');
 
-      const requestDto = {
-        id: '123',
-        nonce: 1,
-        workflowInstanceId: '42',
-        workstepInstanceId: '24',
-        fromSubjectAccountId: fromBpiSubjectAccount.id,
-        toSubjectAccountId: toBpiSubjectAccount.id,
-        payload: 'payload1',
-        signature: 'signature',
-      } as CreateTransactionDto;
-
-      const newTransactionId = await controller.createTransaction(requestDto);
+      const transaction = new Transaction(
+        '123',
```

Comment:
would it make sense we use named params here just for sake of better readability?

Reply:
there is an issue to move hardcoded test values in separate place, lets maybe clean up existing mock storage agents with this work because all other places have these hardcoded values, and then clean up naming and stuff?